beat/config.py

```diff
@@ -681,8 +683,12 @@ def get_hypernames(self):
         for i, wc in enumerate(self.waveforms):
             if wc.include:
                 for c in wc.channels:
-                    hypername = '_'.join(('h', wc.name, str(i), c))
-                    hids.append(hypername)
+                    if wc.freqdomain_include:
```

Comment:
same here then its only if spectral_domain and else, time-domain ...
also only the hypername line would need to be duplicated within the if-else, the append can be after that only once